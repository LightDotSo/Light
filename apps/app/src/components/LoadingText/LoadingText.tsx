import clsx from "clsx";
import type { FC } from "react";

export type LoadingTextProps = {
  className?: string;
};
export const LoadingText: FC<LoadingTextProps> = ({
  className = "h-3 w-6",
}) => {
  return (
    <span
      className={clsx(
        "animate-pulse rounded-full bg-emphasis-medium",
        className,
      )}
    />
  );
};
