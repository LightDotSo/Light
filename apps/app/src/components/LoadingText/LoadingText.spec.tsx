import { expect } from "@jest/globals";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

import { LoadingText } from "./LoadingText";

it("Renders LoadingText", () => {
  const { asFragment } = render(<LoadingText />);
  expect(asFragment()).toMatchSnapshot();
});
