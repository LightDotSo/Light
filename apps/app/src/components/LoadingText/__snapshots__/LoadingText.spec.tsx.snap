// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders LoadingText 1`] = `
<DocumentFragment>
  <span
    class="animate-pulse rounded-full bg-emphasis-medium h-3 w-6"
  />
</DocumentFragment>
`;
